multichannel_timeseries/anaconda-project.yml

```diff
@@ -0,0 +1,89 @@
+name: multichannel_timeseries
+description: Set of examples to demonstrate the display and annotation of multichannel timeseries data in Neuroscience with HoloViz libraries.
```

Comment:
Suggestion as I noticed the text was quite long compared to other ones and affects the main gallery. 
<img width="342" alt="image" src="https://github.com/user-attachments/assets/1fdc0b94-124a-450a-8713-c9a10bc0b5c4">


```suggestion
description:  Display and annotate medium/large multichannel timeseries data in Neuroscience
```

Reply:
changed to "Display and annotate multichannel timeseries data in Neuroscience"